cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/SystemAppTask.java

```diff
@@ -16,42 +16,72 @@
 
 package io.cdap.cdap.internal.app.worker;
 
+import com.google.common.io.ByteStreams;
 import com.google.gson.Gson;
 import com.google.inject.Guice;
+import com.google.inject.Inject;
 import com.google.inject.Injector;
+import io.cdap.cdap.api.artifact.ArtifactId;
 import io.cdap.cdap.api.artifact.CloseableClassLoader;
+import io.cdap.cdap.api.security.store.SecureStore;
 import io.cdap.cdap.api.service.worker.RunnableTask;
 import io.cdap.cdap.api.service.worker.RunnableTaskContext;
 import io.cdap.cdap.api.service.worker.RunnableTaskRequest;
+import io.cdap.cdap.api.service.worker.SystemAppTaskContext;
+import io.cdap.cdap.common.NotFoundException;
+import io.cdap.cdap.common.conf.CConfiguration;
+import io.cdap.cdap.common.conf.Constants;
 import io.cdap.cdap.common.id.Id;
 import io.cdap.cdap.internal.app.runtime.artifact.ArtifactDetail;
+import io.cdap.cdap.internal.app.runtime.artifact.ArtifactManagerFactory;
 import io.cdap.cdap.internal.app.runtime.artifact.ArtifactRepository;
 import io.cdap.cdap.internal.app.runtime.artifact.ArtifactRepositoryReader;
+import io.cdap.cdap.internal.app.runtime.artifact.PluginFinder;
+import io.cdap.cdap.metadata.PreferencesFetcher;
 import io.cdap.cdap.security.impersonation.EntityImpersonator;
 import io.cdap.cdap.security.impersonation.Impersonator;
+import org.apache.twill.discovery.DiscoveryServiceClient;
+import org.apache.twill.filesystem.Location;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
 
 /**
  * SystemAppTask launches a task created by system app with application classloader
  */
 public class SystemAppTask implements RunnableTask {
 
   private static final Gson GSON = new Gson();
+  private CConfiguration cConf;
```

Comment:
final

Reply:
Also, add an empty line between static and non-static fields

Reply:
Done